cmd/kops/create_cluster.go

```diff
@@ -664,7 +667,7 @@ func parseZoneList(s string) []string {
 	return filtered
 }
 
-func supportsPrivateTopology(n *api.NetworkingSpec) bool {
+func hasValidNetwork(n *api.NetworkingSpec) bool {
```

Comment:
Because at first I read it as a preemptive validating for private topology, but because it returns a book it really is only validating 1 thing - I thought about adding the associate public ip logic to it - but quickly realized we would have a list of potential failures.... the rename is left over from that thought experiment - left the name in so I didn't do it again